test/unit/voldemort/server/socket/ClientRequestExecutorPoolTest.java

```diff
@@ -292,11 +292,7 @@ public void testNonExistentHost() throws Exception {
         // JDK 1.6 throws UnresolvedAddressException, which is not even an
         // instance of IOException or ConnectException . JDK 1.8 fixes this
         // issue and uses the ConnectException .
-        try {
-            testConnectionFailure(pool, nonExistentHost, ConnectException.class);
-        } catch(Exception e) {
-            testConnectionFailure(pool, nonExistentHost, UnresolvedAddressException.class);
-        }
+        testConnectionFailure(pool, nonExistentHost, UnresolvedAddressException.class);
```

Comment:
Sidian, can you also remove the comments above it. I was stupid https://github.com/voldemort/voldemort/commit/fa5bf0e2356de94edfa11d18cd7602fe0f78a29b

The exception is same in JDK 1.6 and 1.8 , but some one registered the old address which caused the exception to be different. 

This fixed it, but my regression broke it.
https://github.com/voldemort/voldemort/commit/b46c97faf9a079961cd8b3dc701138b1ffc8c9dd